linux_os/guide/system/software/integrity/fips/system_boot_in_fips_mode/rule.yml

```
@@ -0,0 +1,55 @@
+documentation_complete: true
+
+title: 'Verify that the system was booted with fips=1'
+
+description: |-
+    On a system where FIPS 14032 mode is enabled, the system must be booted with the
```

```suggestion
    On a system where FIPS 140-2 mode is enabled, the system must be booted with the
```

```suggestion
    On a system where FIPS 140-3 mode is enabled, the system must be booted with the
```

Which one is the correct one?

---

140-3

---

In crypto's docs, we prefer to use "FIPS 140" now, since there may be a FIPS 140-4 a lot sooner than the time it took to go from -2 to -3.